fs/operations/dedupe.go

```diff
@@ -209,6 +209,20 @@ func dedupeMergeDuplicateDirs(f fs.Fs, duplicateDirs [][]fs.Directory) error {
 	}
 	for _, dirs := range duplicateDirs {
 		if !fs.Config.DryRun {
+
+			//Find the largest directory and merge duplicates into it
+			largestDir := dirs[0]
+			largestDirId := 0
+			counter := 0
+			for _, dir := range dirs {
+				if dir.Size() > largestDir.Size() {
```

Comment:
Unfortunately there aren't many backends which support the `Size()` attribute and `drive` isn't one of them :-(

I think you'll find they are all -1

So I think doing a f.List(dir.Remote()) will be necessary to find the size.